test/test_torch.py

```diff
@@ -7392,6 +7392,75 @@ def run_test(M, sign=1):
         M = random_fullrank_matrix_distinct_singular_value(3, 2, 3, dtype=dtype, device=device)
         run_test(M, sign=-1)
 
+    @skipCUDAIfNoMagma
+    @skipCPUIfNoLapack
+    @dtypes(torch.float, torch.double)
+    def test_matrix_exp(self, device, dtype):
```

Comment:
Thank you! Do you think it makes sense to include it? I just thought that comparison to the analytic solution should be sufficient.

Reply:
I think it would be nice to have it for the non-trivial approximants, that is, T_8, T_12 and T_18, just to make sure that they match the expected result, otherwise it might happen that some of that code is never tested.